src/meetingreadinesschecker/MeetingReadinessCheckerConfiguration.ts

```diff
@@ -16,4 +16,14 @@ export default class MeetingReadinessCheckerConfiguration {
    * If null, it will use the default value.
    */
   waitDurationMs: number = 3000;
+
+  /**
+   * Specify the frequency of oscillation in hertz for the audio output test.
+   */
+  audioOutputFrequency?: number = 440;
```

Comment:
Do we need optional `?` if we already use default value?

Reply:
Without `?`, TS throws
> Property 'audioOutputFrequency' is missing in type 'Child' but required in type 'Parent'.

```
class Parent {
  name: string = 'hey';
  age: number = 12
  audioOutputFrequency = 440;
}

class Child implements Parent {
  name = 'Name';
  age = 99;
}
```


I don't think SDK users implement this configuration, but using `?` ensures backward compatibility.

Reply:
Interesting, I thought implements only use for interface :)